test/extended/gssapi.sh

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env bash
+#
+# Extended tests for logging in using GSSAPI
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+STARTTIME="$(date +%s)"
+
+PROJECT='gssapiproxy'
```

Comment:
vars that are not created to be exported or passed to other scripts should not be in caps